eosclubhouse/utils.py

```diff
@@ -39,7 +39,7 @@
 
 
 def get_alternative_quests_dir():
-    return os.path.join(GLib.get_user_data_dir(), 'quests')
+    return os.path.join(GLib.get_home_dir(), 'quests')
```

Comment:
I don't like the idea of having this folder in the user home. It could simplify the content creation for the user, but the app loose the control over that folder, because we don't know what the user can add there.